### PR TITLE
Add to autocomplete words split off

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/admin/autocomplete.py [hunk 1]
```diff
@@ -145,14 +145,23 @@ def get_query_filters(self, queryset, query, search_fields):
...
         """
-        for value in QUERY_REGEX.split(query):
-            if value:
-                query_filters = [
-                    Q(**{'{}__icontains'.format(field): value})
-                    for field in search_fields
-                ]
-                queryset = queryset.filter(reduce(operator.or_, query_filters))
-
+        is_split = getattr(self.model, 'autocomplete_words_split', False)
```
Comment (mkurek) [inline src/ralph/admin/autocomplete.py]: `is_split` -> `split_by_words`

File: src/ralph/admin/autocomplete.py [hunk 2]
```diff
@@ -145,14 +145,23 @@ def get_query_filters(self, queryset, query, search_fields):
...
+                    ]
+                    queryset = queryset.filter(
+                        reduce(operator.or_, query_filters)
+                    )
+        else:
+            query_filters = [
+                Q(**{'{}__icontains'.format(field): query})
+                for field in search_fields
+            ]
+            queryset = queryset.filter(reduce(operator.or_, query_filters))
```
Comment (mkurek) [inline src/ralph/admin/autocomplete.py]: this line is common between if and else - move it one level behind

**Comment by mkurek** [comment]
what about unit tests? :)

**Comment by ar4s** [comment]
:ok:

File: src/ralph/admin/tests/tests_autocomplete.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    def test_or(self):
+        autocomplete_list = AutocompleteList()
+        autocomplete_list.model = RalphUser
+        result = autocomplete_list.get_query_filters(
+            RalphUser.objects.all(),
+            'first_name_1 last_name_1',
+            ['username', 'first_name', 'last_name']
+        )
+        self.assertEqual(result.first().id, self.user.id)
```
Comment (mkurek) [inline src/ralph/admin/tests/tests_autocomplete.py]: `self.assertEqual(list(result), [self.user])`

File: src/ralph/admin/tests/tests_autocomplete.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+    def test_not_or(self):
+        autocomplete_list = AutocompleteList()
+        autocomplete_list.model = Region
+        result = autocomplete_list.get_query_filters(
+            Region.objects.all(),
+            'pl',
+            ['name']
+        )
+        self.assertEqual(result.first().id, self.region.id)
```
Comment (mkurek) [inline src/ralph/admin/tests/tests_autocomplete.py]: `self.assertEqual(list(result), [self.region])`

**Comment by mkurek** [comment]
:ok:

**Pull request merged**